checkout-ui-custom/checkout6-custom.js

```diff
@@ -94,7 +94,8 @@ class checkEmailAuthConflict {
 
 	init() {
 		const _this = this;
-		$(window).one('orderFormUpdated.vtex', function(evt, orderForm) {
+		$(window).one('orderFormUpdated.vtex', function(_, orderForm) {
+			if (orderForm.userType === 'callCenterOperator') return
```

Comment:
Should this be added to the `$(window).on('authenticatedUser.vtexid closed.vtexid'` function below as well?

Reply:
yes!, updated!